reference/faq/known-issues.md

```diff
@@ -14,6 +14,32 @@ If you want to delete your project and cancel your subscription, simply go to yo
 
 This will delete your project and stop invoicing for this project. If you have multiple projects, your subscription will continue until you don't have any projects left.
 
+## How can I export my data?
+
+If you have file mounts, you can keep a copy of the files using `rsync` like below.
+
+```
+rsync -r <project-id>-<environment>@ssh.<region>.platform.sh:/app/private/ .
+```
+
+Data stored in database can be dumped onto the filesystem and then `rsync` to local as well.
```

Comment:
You can also dump it directly to local, streaming over SSH, with just:

```
platform sql-dump
```

That should work for MySQL or PostgreSQL